app/src/main/java/org/gem/indo/dooit/views/web/MinimalWebViewActivity.java

```diff
@@ -92,8 +96,10 @@ public void onProgressChanged(WebView view, int progress) {
                 }
             }
         });
-
-        webView.loadUrl(getIntent().getStringExtra(INTENT_URL));
+        Map<String, String> headers = new HashMap<String,String>();
+        headers.put("Accept-Language","id");
```

Comment:
Please don't hardcode this header. The app has to support multiple languages, so the language code has to be determined by the phone. See [DooitManager](https://github.com/praekeltfoundation/gem-bbb-indo/blob/4616a475f0af4984b699e7dd9d8a0092a3e56c06/app/src/main/java/org/gem/indo/dooit/api/managers/DooitManager.java#L73) for an example.